composeApp/src/commonMain/kotlin/com/sixbynine/transit/path/widget/WidgetDataFetcher.kt

```diff
@@ -98,7 +102,7 @@ object WidgetDataFetcher {
         stationLimit: Int,
         stations: List<Station>,
         lines: Collection<Line>,
-        sort: StationSort,
+        sort: StationSort?,
```

Comment:
I'd just keep this non-null and pass `Alphabetical` where you're calling it for the commute widget. You're passing station limit 1 so it doesn't matter, but it's not invalid.

Allowing null here creates a potential later problem where someone calls this with a null sort and station limit greater than 1